src/Extension/Workflow/InteractionAnswerBasedTransitionResolver.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\Workflow;
+
+use SprykerSdk\Sdk\Core\Application\Dto\ReceiverValue;
+use SprykerSdk\Sdk\Extension\Exception\UniqueValueException;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\ValueReceiver\ValueReceiverInterface;
+use SprykerSdk\SdkContracts\Workflow\TransitionResolverInterface;
+
+class InteractionAnswerBasedTransitionResolver implements TransitionResolverInterface
+{
+    /**
+     * @var string
+     */
+    protected const RESOLVER_NAME = 'interactive';
+
+    /**
+     * @var string
+     */
+    public const QUESTION = 'question';
```

Comment:
Public const should be above protected

Reply:
Why? It's keys for structure I use it outside of the class

Reply:
it is a common structure for class - public -> protected -> private. Please put public above protected